media-libs/opensubdiv/patches/opensubdiv-3.4.0.patchset

```diff
@@ -1,44 +0,0 @@
-From 0bd024622a84da627e47b0fbc0da26359f7e2e79 Mon Sep 17 00:00:00 2001
-From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
-Date: Sun, 10 Dec 2017 11:27:28 +0100
-Subject: Platform fixes
-
-
-diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 3920896..254a4db 100644
---- a/CMakeLists.txt
-+++ b/CMakeLists.txt
-@@ -337,7 +337,7 @@ endif()
- if(NOT NO_CUDA)
-     find_package(CUDA 4.0)
- endif()
--if(NOT NO_GLFW AND NOT NO_OPENGL AND NOT ANDROID AND NOT IOS)
-+if(NOT NO_GLFW AND NOT NO_OPENGL AND NOT ANDROID AND NOT IOS AND NOT HAIKU)
-     find_package(GLFW 3.0.0)
```

Comment:
GLFW is definetely not available on Haiku...

Reply:
cmake checks if it's available, if not it sets a flag for it, doesn't bail out on it